.scripts/linux/pipeline/build-images-core.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# The version we're building, defaults to dev
+VERSION=0.0.0-devel
+CLOUD_PROVIDER=local
+
+# Read the version flag if given
+num=0
+for var in "$@"
+do
+    eval flag="\$$num"
+
+    if [[ $flag == "--provider" ]]; then
+        ((num=num+1))
+        eval CLOUD_PROVIDER="\$$num"
+    fi
+
+    if [[ $flag == "--version" ]]; then
+        ((num=num+1))
+        eval VERSION="\$$num"
+    fi
+
+    ((num=num+1))
+
+done
+
+# Define our base images
+IMG_HTTP_SERVER="tensorworks/sps-http-server"
```

Comment:
we are changing our container name from `tensorworks/sps-http-server` to `tensorworks/sps-frontend`

Reply:
This has been resolved in 464f644b245d0fb979821901c6096b494af6c9fc and 4639b436548d7e2d396564027d16d4d75e284482